include/igl/arap.cpp

```diff
@@ -22,20 +22,24 @@
 #include <cassert>
 #include <iostream>
 
+template <typename Scalar>
+using MatrixXX = Eigen::Matrix<Scalar, Eigen::Dynamic, Eigen::Dynamic>;
```

Comment:
Note that this is only necessary because libigl officially supports Eigen version 3.3.7. I'm not sure from which version, but in 3.4.0, Eigen has a typedef already for MatrixX. Then, this should be removed in favour of the Eigen defined typedef.

Reply:
good to know!